doc/source/user/absolute_beginners.rst

```diff
@@ -826,6 +831,7 @@ deviation, and more. ::
   3.0
 
 .. image:: images/np_aggregation.png
+  :alt: <Alternative text here>
```

Comment:
array([1, 2, 3.])?

Reply:
Three array elements of array([1, 2, 3.). The first column has data.max() applied with a result of 3. The second column has data.min() applied with a result a result of 1. The third column has data.sum() with a result of 6?

Reply:
This is a nice description, but I initially was confused by the "columns of array data" because the "column" actually refers to the image, not the array in the image!

Instead of the word "column", maybe something like:

> A three-panel image where the first panel shows the minimum of an array with three elements, the second shows the maximum, and the third shows the sum

Reply:
In the future, we could compile a styling guideline for alt-text, just like there's a [style guide for numpy documentation ](https://numpy.org/doc/stable/docs/howto_document.html)(such as the guideline for pluralize matrix as matrices)

Reply:
There are three arrays of array(1, 2, 3]. The maximum of an array using data.max() is 3. The minimum of an array using data.min() is 1. The sum of an array with three elements using data.sum if 6.